includes/admin/meta-boxes/views/html-related-orders-row.php

```diff
@@ -11,7 +11,7 @@
 ?>
 <tr>
 	<td>
-		<a href="<?php echo esc_url( get_edit_post_link( $row_order->get_id() ) ); ?>">
+		<a href="<?php echo esc_url( $row_order->get_edit_order_url() ); ?>">
```

Comment:
This doesn't render correct URLs for related Subscriptions when on the Edit Order screen.

This may or may not be the correct API to use when/if https://github.com/woocommerce/woocommerce/issues/35601 is implemented.

Reply:
> This doesn't render correct URLs for related Subscriptions when on the Edit Order screen.

This is fine IMO. There's no correct URL for viewing subscriptions in HPOS yet until #223 is implemented so I've opened a new issue to address this: https://github.com/Automattic/woocommerce-subscriptions-core/issues/277